### DISCUSSION TITLE
Custom key binds doesn't work in Editor mode

### DISCUSSION THREAD

**Discussion by DeFaLT-dj** [Mojang/minecraft-editor#166] [Bugs]
I am not 100% sure if this can be considered as a bug or if it is intended behavior but I couldn't find any similar report so I decided to fill one. In Minecraft, I usually prefer to use SHIFT for sprint and CTRL for crouch which is, by default, other way around so I just change them in the settings. Today I decided to give Minecraft's editor a try but one of the first things I noticed is I can't move down and changing the key binds to default fixed the issue.
 
**Repro steps**
1. Set a custom key bind for crouch and sprint actions
2. Try to use custom crouch button to move lower while in editing mode "aka when you hold right click to move"

**Observed result vs. what you actually expected to happen**
It would make sense if it would work with your regular key binds but it does not. I will attach a OneDrive link which includes a [video](https://1drv.ms/v/s!AtpfJBLal6aQg98Z2rK1u9XEKsG4qg?e=8JyTaA) demonstration (I couldn't upload it here because of file size limits)

**Other helpful info:**

- Preview build version: 1.20.80.20
- Frequency: Always
- Local single-player
- Graphics card: DX12 
- Device info: laptop Lenovo Legion

**Comment by m-brisch** [comment]
@DeFaLT-dj it should be using the keybinds you have set for Fly Up/Fly Down.

For example if I change mine to use `[` and `]` then those change my elevation in Editor when holding down right click:
![image](https://github.com/Mojang/minecraft-editor/assets/29804144/1b4a8d28-948e-45fc-85fa-d10b2c3bbb86)

Let me know if you're still seeing issues!
